### PR TITLE
Resolved import issue and python3 print problem.

### PR DESCRIPTION
**Pull request by oldmerkum**
In ann4brains/nets.py must import local modules using from ann4brains.<module>
In ann4brains/optimizers.py print function use was not updated for python3, fixed.

### CODE REVIEW

File: ann4brains/optimizers.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def caffe_SGD(solver_filename, niter, test_interval, test_iter,
     # num_metrics = solver.net.blobs[pred_layer_id].data.shape[1] + 1
 
     if start_weights_name != None:
-        print 'starting from: ' + start_weights_name
+        print('starting from: ', start_weights_name)
```
Comment (jeremykawahara) [inline ann4brains/optimizers.py]: Good catch thanks!

File: examples/proto/hello_world_deploy.prototxt [hunk 2]
```diff
@@ -17,12 +17,12 @@ layer {
   bottom: "data"
   top: "e2n"
   param {
-    lr_mult: 1
-    decay_mult: 1
+    lr_mult: 1.0
```
Comment (jeremykawahara) [inline examples/proto/hello_world_deploy.prototxt]: Are these changes necessary?

File: examples/proto/hello_world_deploy.prototxt [hunk 3]
```diff
@@ -53,7 +53,7 @@ layer {
   bottom: "e2n"
   top: "e2n"
   relu_param {
-    negative_slope: 0.33
+    negative_slope: 0.33000001311302185
```
Comment (jeremykawahara) [inline examples/proto/hello_world_deploy.prototxt]: Needed?

File: examples/proto/hello_world_solver.prototxt [hunk 4]
```diff
@@ -2,13 +2,13 @@ train_net: "./proto/hello_world_train.prototxt"
 test_net: "./proto/hello_world_test.prototxt"
 test_iter: 100
 test_interval: 100
-base_lr: 0.01
+base_lr: 0.009999999776482582
```
Comment (jeremykawahara) [inline examples/proto/hello_world_solver.prototxt]: Needed?

File: ann4brains/nets.py [hunk 5]
```diff
@@ -2,17 +2,17 @@
 import numpy as np
 import os
 import matplotlib.pyplot as plt
-import cPickle
+import pickle as cPickle
```
Comment (jeremykawahara) [inline ann4brains/nets.py]: Great fix thanks!

**Review by jeremykawahara — COMMENTED** [review]
Thanks for your PR @oldmerkum !
The imports and print fixes make sense to me. I'm wondering about the other files though. Are these changes needed? The binary files I also don't think should be checked in. Could you only include in this PR the imports and print fixes?

**Comment by oldmerkum** [comment]
Those were most likely generated from running the helloworld.py test.
I will remove them and resubmit. Thanks for replying.

**Review by oldmerkum — COMMENTED** [review]
I removed the un-needed changes.

**Review by jeremykawahara — APPROVED** [review]
Looks great! Thank you @oldmerkum!
